libraries/AP_Scripting/drivers/SIM7600.md

```diff
@@ -38,38 +52,41 @@ serial devices.
 
 This is the first octet of the server IP address to connect to. The
 full IP address is constructed from LTE_SERVER_IP0 through
-LTE_SERVER_IP3. Range: 0-255.
+LTE_SERVER_IP3. Range: 0-255. This is not used with PPP.
 
 ## LTE_SERVER_IP1
 
 This is the second octet of the server IP address to connect to.
-Range: 0-255.
+Range: 0-255. This is not used with PPP.
 
 ## LTE_SERVER_IP2
 
 This is the third octet of the server IP address to connect to.
-Range: 0-255.
+Range: 0-255. This is not used with PPP.
 
 ## LTE_SERVER_IP3
 
 This is the fourth octet of the server IP address to connect to.
-Range: 0-255.
+Range: 0-255. This is not used with PPP.
 
 ## LTE_SERVER_PORT
 
 This sets the IPv4 port of the server to connect to. This should match
 the port that your ground control station or server is listening on.
-Range: 1-65525. Default: 20001.
+Range: 1-65525. This is not used with PPP.
 
 ## LTE_BAUD
 
 This sets the baud rate for the serial port to the LTE modem. Common
 values are 9600, 57600, or 115200. The modem must be configured to use
-the same baud rate. Range: 9600-115200. Default: 115200.
+the same baud rate. Range: 9600-3686400. Default: 115200.
 
 If using something other than 115200 you need to connect to the modem
 with a terminal program and use AT+IPREX=BAUD to set the baud rate and
-save with AT&W
+then save with AT&W. If the modem is wired to a flight controller then
+you can use the SERIAL_PASS parameters to give temporary control of
+the modem to a USB port so you can use a terminal protocol to
+configure the modem.
```

Comment:
Why can't the Lua script automatically just set the baud? Future update?

Reply:
yes, should be possible, just complex for the recovery with +++